File: main/src/main/java/net/fwitz/math/main/binary/escape/newton/NewtonsMethodSin.java
package net.fwitz.math.main.binary.escape.newton;

import net.fwitz.math.binary.complex.Complex;

public class NewtonsMethodSin {
    public static void main(String[] args) {
        NewtonsMethodPlot.render("sin(z) (Newton's Method)",
                Complex::sin,
                Complex::cos);
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/escape/root/newton/NewtonsMethodSin.kt
package net.fwitz.math.main.binary.escape.root.newton

import net.fwitz.math.binary.complex.Complex

object NewtonsMethodSin {
    @JvmStatic
    fun main(args: Array<String>) = NewtonsMethodPlot.render(
        name = "sin(z) (Newton's Method)",
        f = Complex::sin,
        df = Complex::cos
    )
}